### DISCUSSION TITLE
Is there a way to merge the upstream repository tag into the branch created on fork using Github API

### DISCUSSION THREAD

**Discussion by pm-pramod-pisal** [community/community#24900] [Apps, API and Webhooks]
<p>Is there any way to merge the tag from upstream repository to the branch created on fork.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/pm-pramod-pisal">@pm-pramod-pisal</a>,</p>
<p>It’s not possible to merge a specific tag into a branch. However, it’s possible to merge a pull request using this endpoint: <a href="https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button" rel="nofollow noopener">https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button</a>.  </p>
<p>I hope this helps!</p>
<p>Andrea</p>
